docs/src/devdoc/own_module.rst

```diff
@@ -21,3 +21,53 @@ Finally, also provide meaningful tests for your module in ``test/modules``.
 For further questions feel free to ask us on our Discord_ server.
 
 .. _`Discord`: https://discord.gg/mnrEQWVAed
+
+
+Observable averaging and error estimation
+-----------------------------------------
+
+When writing modules for MAICoS you will collect per-frame observables in the
+``self._obs`` object inside your :meth:`_single_frame` method. Each entry in
+``self._obs`` should be a numeric scalar or a numpy array. Make sure to keep
+the shape of any array-valued observables consistent across frames. (For example
+by using the bin numbers provided by the Base classes.)
+
+On the first analysed frame MAICoS initialises several internal accumulators
+from ``self._obs``:
+
+- ``sums``: running sums of the observables (used for totals or mean accumulation)
+- ``means``: running means (initialised from the first-frame values)
+- ``sems``: standard errors of the mean (initially zero or NaN arrays with the correct
+  shape)
+- ``pop``: integer counts per observable/bin
+- ``M2``: running M2 accumulator used for variance/error estimation
```

Comment:
maybe but the as `:attr:` syntax to format this in a nice way as real attributes.